core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingModalService.js

```diff
@@ -14,13 +14,21 @@
 
 /**
  * @fileoverview Service which handles opening and closing
- * the training modal used for both unresolved answers.
+ * the training modal used for unresolved answers.
  */
 
 oppia.factory('TrainingModalService', [
   '$rootScope', '$uibModal', 'AlertsService', 'UrlInterpolationService',
   function($rootScope, $uibModal, AlertsService, UrlInterpolationService) {
     return {
+      /**
+      * Opens unresolved answer trainer modal for given answer.
+      * @param {Object} - unhandledAnswer The answer to be trained.
+      * @param {Boolean} externalSave - Whether to save the modified training
+          data externally in state.
```

Comment:
Let's add a bit more context here. Why wouldn't we want to do this? (i.e. why wouldn't it be true)?

Reply:
I don't know. As long as I see, it would always be true but it was present in earlier directives and so I thought maybe it has some significance. Should I remove this argument?

Reply:
Done